tests/acceptance/features/bootstrap/FeatureContext.php

```diff
@@ -2944,6 +2944,25 @@ public function theContentInTheRespShouldMatchWithFileInTheServerRoot(string $pa
 		$this->theFileWithContentShouldExistInTheServerRoot($path, $content);
 	}
 
+	/**
+	 * @Then the content in the response should match the following content:
+	 *
+	 * @param PyStringNode $content
+	 *
+	 * @return void
+	 */
+	public function theContentOfTheRequestedFileInTheResponseShouldBe(PyStringNode $content):void {
+		$actualContent = $this->getResponse()->getBody()->getContents();
+		$pattern = ["/--\w*/", "/\s*/m"];
+		$actualContent = \preg_replace($pattern, "", $actualContent);
+		$expectedString = \preg_replace($pattern, "", $content->getRaw());
+		Assert::assertSame(
+			$expectedString,
+			$actualContent,
+			"The content of the file does not match with '$expectedString'"
+		);
+	}
+
```

Comment:
Move this to `WebDav.php` and it can be used as below.
```suggestion
	/**
	 * @Then the content in the response should match the following content:
	 *
	 * @param PyStringNode $content
	 *
	 * @return void
	 */
	public function theContentInTheResponseShouldMatchTheFollowingContent(PyStringNode $content): void {
		$this->checkDownloadedContentMatches($content->getRaw())
	}

```